SCClassLibrary/Common/Math/Date.sc

```diff
@@ -74,4 +107,22 @@ Date {
 		_prStrFTime;
 		^this.primitiveFailed
 	}
+	< { arg other;
+		^this.rawSeconds < other.rawSeconds
+	}
+	<= { arg other;
+		^this.rawSeconds <= other.rawSeconds
+	}
+	> { arg other;
+		^this.rawSeconds > other.rawSeconds
+	}
+	>= { arg other;
+		^this.rawSeconds >= other.rawSeconds
+	}
+	== { arg other;
```

Comment:
If we have an `==` method, we also need a `hash`method.
```
hash {
^this.instVarHash([\rawSeconds])
}
```

Reply:
Okay, added this...thanks!